infra/templates/ado-bootstrap-iac-pipeline/README.md

```diff
@@ -0,0 +1,193 @@
+This provides a starting point for anyone interested in having an Azure DevOps
+Project that will 'run' a target Cobalt template whenever that template changes.
+
+## Intended audience
+
+*FIXME:* Application developer that is brand new to Cobalt templating and it's *Cobalt Infrastructure Template* (CIT) developer workflow.
+
+### An example
+Imagine that you are building a new set of web applications which are destined
+to run on Azure. Those applications are going to use a few runtime resources, and you've
+wisely decided to use a Cobalt template to satisfy and centralize your Infrastructure as Code ("IaC") 
+needs across all of those applications. 
+
+To get started, you want a central Git repository to store your IaC in, 
+and following [solid engineering fundamentals](https://github.com/microsoft/code-with-engineering-playbook/tree/master/continuous-integration#continuous-integration),
+you've decided to wrap CI/CD, and staged deployments around your IaC to help
+protect the quality of your infrastructure changes.
+
+This bootstrap (itself a Cobalt template) can be used to quickly, safely, and easily create 
+the ADO resources (a Project, Git Repository, Build Definitions, Variable Groups, 
+Service Connections, etc) that you'll need in that example.
+
+### Terminology
+Given that we're using a Cobalt template to create resources that will then operate other
+Cobalt templates, discussing matters can become a bit confusing. To help with that, and 
+to get specific about a few other terms we'll use, let's define a few terms and then use
+them consistently, within this document and this bootstrap template itself.
+
+*FIXME:* during final edits, make sure this list is concise and sufficient.
+
+|Term|Meaning|
+|----|----|
+|ADO|Azure DevOps, AKA "AzDO," elsewhere|
+|Application(s)|The application that will rely on the resources managed by the Infrastructure Template|
+|Bootstrap Template|This Cobalt template, as described in the example, above|
+|IaC|Infrastructure as Code|
+|Infrastructure Template|Whichever Cobalt template that will manage the Application(s) runtime resources. _By default, the Bootstrap Template points to the 'Hello World' example template in the official Cobalt GitHub repository_|
+|Project|The ADO Project that the Bootstrap Template will add pipeline resources to|
+|IaC Repository|(empty) Git Repository in the Project where the Infrastructure Template is expected to be kept|
+
+## What the Bootstrap Template needs, provisions, and outputs
+
+To work properly, the Bootstrap Template needs some information about how you want things named, how many environments
+are desired (and their details), and little else. The template also creates many kinds of resources
+
+### Decide with Project to use
+
+Visit your ADO Organization site and select or create a project. This project will house the pipelines that
+will operate your IaC CI/CD pipelines. Note that project's name. We'll need to use it as an input, in the next section.
```

Comment:
May be worthwhile to point out that if the user wants to create the project as well, then they can modify the template to do so. perhaps the implementation of that might live in a comment alongside the project `data` source?